application/testing/CMakeLists.txt

```diff
@@ -230,6 +230,7 @@ f3d_test(NAME TestTextures DATA WaterBottle.glb ARGS --texture-material=${F3D_SO
 f3d_test(NAME TestMetaDataImporter DATA BoxAnimated.gltf ARGS -m UI)
 f3d_test(NAME TestMultiblockMetaData DATA mb.vtm ARGS -m UI)
 f3d_test(NAME TestMultiFileMetaData DATA mb/recursive ARGS -m --multi-file-mode=all UI)
+f3d_test(NAME TestMultiFileModeDir DATA mb ARGS -n --multi-file-mode=dir UI)
```

Comment:
do we need a "group-index" CLI option ? Currently there is no way to load a specific group on startup.

Reply:
The group that shows on startup should be the one that holds the latest of the input files. Or at least that's what I assumed based on #756.

However it looks like `AddFile`'s return value is ignored in places. Once that is fixed the there will be a way to select a specific group without needing an index option

Reply:
> The group that shows on startup should be the one that holds the latest of the input files.

I agree

> without needing an index option

What do you mean ?

Reply:
you would be able to do `f3d foo/ foo/bar.xyz` and be sure that the group containing `foo/bar.xyz` would be shown regardless of what the grouping logic is and what the position of `foo/bar.xyz` is in the recursive walk of the directory

Reply:
I was mistaken, the files passed as argument are supposed to be shown in order with the _first_ one displaying on startup.

So, without changing that behavior, it is possible to load a specific group by picking the _first_ of the arguments, for example:
- `f3d --multi-file-mode=dir testing/data/mb/mb_0_0.vtu testing/data/mb -n` ![f3d-1](https://github.com/user-attachments/assets/5c15335f-fc0d-463d-bce5-95cbfda58e93)
- `f3d --multi-file-mode=dir testing/data/mb/recursive/mb_0_0.vtu testing/data/mb -n` ![f3d-2](https://github.com/user-attachments/assets/859ec9fc-8f8d-4aed-8f8c-b1c2b1187f26)

Reply:
Ok! As long as its consistent fine by me! My original question (do we need a CLI option to select the group to load?) is still valid but its out of scope of this PR entirely anyway :)